circle.yml

```diff
@@ -1,7 +1,7 @@
 ---
 machine:
   ruby:
-    version: '2.3.3'
+    version: '2.4.1'
```

Comment:
I'd love to get a test matrix for 2.3 and 2.4 branches of ruby.

Otherwise I'd be fine to simply drop 2.3 support. People who need it can use older versions of mutant.